pipeline/src/graph-queries/addressables/visualStories.ts

```diff
@@ -0,0 +1,14 @@
+const query = `
+  visual-stories {
+    title
+    promo {
+      ...on editorialImage {
+        non-repeat {
+          caption
+        }
+      }
+    }
+  }
+`;
```

Comment:
[In the table](https://www.notion.so/wellcometrust/All-search-definition-acceptance-criteria-1366687658a180f28254d4429bf764ce) we say we could use standfirst, but I know in your findings you opted for promo text over it. Should it still be used for query?

Reply:
I think not. I'll update the table.